birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything
```

Comment:
Can you tell me why this file is needed?

Reply:
Could you write a comment about that? These generic comments does not really help.

Reply:
Documentation is in progress for the entire bundle, which will be available in few hrs, I hope...